config/settings.yml

```diff
@@ -40,3 +40,5 @@ workflow_services_url: ''
 c2m_sql_limit: 1000
 
 checksum_algos: ['md5'] # 'sha1' 'sha256'
+
+zip_storage: '' # empty here, override in #{RAILS_ENV}.yml
```

Comment:
actually, might I suggest "tmp"  or something for a local setting, but keep the second half of your comment?  Then we can use it for tests.